packages/istanbul-report-modern-html/LICENSE

```diff
@@ -0,0 +1,24 @@
+Copyright 2012-2015 Yahoo! Inc.
```

Comment:
?

Reply:
that's a blast from the past isn't it :stuck_out_tongue: the original Istanbul library grew out of work in Yahoo; If you're doing a rewrite of the HTML report, let's license as MIT or ISC, to "Contributors".